docs/badge-status.md

```diff
@@ -0,0 +1,56 @@
+# Badges for modules
+
+You can tell at a glance the stage of development and approval of a IBM Cloud Terraform module by checking the status badge in the main readme file in the repository. You can also see the modules that are publicly available in the IBM Cloud Catalog.
+
+## Status badges
+The readme file for the module in GitHub includes a status badge.
+
+### Active statuses
+The following status badges are supported for active modules. The order follows the typical development stages for modules.
+
+<ol>
+  <li>
+    <dl>
+      <dt><img src="https://img.shields.io/badge/Status-Incubating%20(Not%20yet%20consumable)-red?style=plastic" alt="Incubating"></dt>
+      <dd>The module is in early stage and is not yet consumable.</dd>
+    </dl>
+  </li>
+  <li>
+    <dl>
+      <dt><img src="https://img.shields.io/badge/Status-Implemented%20(No%20quality%20checks)-yellowgreen?style=plastic" alt="Stable"></dt>
+      <dd>The module is in advanced development, which indicates that it is valuable to consumers. The module does not yet have quality checks in place.</dd>
+    </dl>
+  </li>
+  <li>
+    <dl>
+      <dt><img src="https://img.shields.io/badge/Status-Stable%20(With%20quality%20checks)-green?style=plastic" alt="Stable (With quality checks)"></dt>
+      <dd>The module is reviewed and approved by the project maintainers. Quality checks are run against the code. The module is not yet supported by IBM</dd>
```

Comment:
I would change "The module is not yet supported by IBM" to "This module is supported by the community". The "community" may be made by IBMers here. The main distinction between stable and graduated is the level of official support. For instance SLZ is being onboarded into the IBM Cloud Support flow.